reflex-dom-core/src/Reflex/Dom/WebSocket.hs

```diff
@@ -8,7 +8,6 @@
 {-# LANGUAGE ForeignFunctionInterface #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE JavaScriptFFI #-}
```

Comment:
Started getting an unknown extension error in 9.8. Doesn't seem to be needed?

Reply:
Ah, looks like this extension became unsupported in all non-JS backends as of 9.8